.grunt-config/widgets-css.js

```diff
@@ -130,24 +131,30 @@ class WidgetsCss {
 	getModulesFrontendScssFiles( filePath, frontendScssFiles = [] ) {
 		fs.readdirSync( filePath ).forEach( ( fileName ) => {
 			const fileFullPath = path.join( filePath, fileName );
+			const isFrontendScssFile = fileName.indexOf( 'frontend.scss' ) > -1;
+			const isWidgetsScssFile = fileName.indexOf( '.scss' ) > -1 && filePath.indexOf( '/widgets' ) > -1;
```

Comment:
@louiswol94 I have added support to include all files inside the /widgets folder here.

Wdyt?

Reply:
@hein-obox Rami said that he prefers if we look for a file name like `widget-*.scss`, instead of a `widgets` folder?

Reply:
@rami-elementor @ramiy @louiswol94 
I prefer to keep the separate widget files inside the widgets folder. I find a bit chaotic to place all the scss files in the scss folder, especially for modules that have over 10 scss files.

Wdyt?